packaging/debian/keybase.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Version=1.0
+Name=Keybase
+Exec=/usr/bin/run_keybase.sh
+Terminal=false
+Type=Application
+Categories=Network;
```

Comment:
Let's kill this file for now, but in future we'll want a visible desktop entry that just restarts the GUI but leaves the service alone so that your KBFS mounts don't disappear under you.